tests/test_classification.py

```diff
@@ -458,3 +458,12 @@ def test_class_distinctiveness(n_classes, metric_mean, metric_dist, exponent,
     assert isinstance(class_dis, float), "Unexpected object of class_dis"
     assert isinstance(num, float), "Unexpected object of num"
     assert isinstance(denom, float), "Unexpected object of denum"
+
+
+@pytest.mark.parametrize("classif", [MDM, FgMDM, MeanField])
```

Comment:
My next PR was to add this test ;-)

To follow the framework of current tests, can you add at line 127 this function?
```
def clf_fittransform(classif, mats, labels):
    clf = classif()
    transf = clf.fit_transform(mats, labels)
    transf2 = clf.fit(mats, labels).transform(mats)
    assert_array_equal(transf, transf2)
```
and then, update line 78 with:
```
    if classif in (MDM, FgMDM, MeanField):
        clf_transform(classif, mats, labels)
        clf_fittransform(classif, mats, labels)
```